app/src/main/res/drawable/ic_left_arrow_24dp.xml

```diff
@@ -0,0 +1,4 @@
+<vector android:height="24dp" android:viewportHeight="400.004"
```

Comment:
WARNING: [UnusedResources] The resource `R.drawable.ic_left_arrow_24dp` appears to be unused
 Unused resources make applications larger and slow down builds.